### PR TITLE
add support for crux linux

### PR DESCRIPTION
**Pull request by alexmat**
Tested / working on my system.

### CODE REVIEW

File: cmd/linuxdb.yaml [hunk 1]
```diff
@@ -290,7 +290,7 @@ distributions:
           - name: gcc
             help: Please install with `sudo zypper in gcc-c++` and try again
           - name: pkg-config
-            help: Please install with `sudo zypper in pkgconf-pkg-config` and try again
+            help: Please install with `sudo zypper in pkg-config` and try again
```
Comment (leaanthony) [inline cmd/linuxdb.yaml:293]: Why was this changed out of interest?

File: cmd/linuxdb.yaml [hunk 2]
```diff
@@ -307,3 +307,22 @@ distributions:
...
+      releases:
+        default:
+          version: default
+          name: Crux Linux
+          gccversioncommand: *gccdumpversion
+          programs:
+            - name: gcc
+              help: Please install with `sudo prt-get depinst gcc-c++ make` and try again
+            - name: pkg-config
+              help: Please install with `sudo prt-get depinst pkgconf-pkg-config` and try again
```
Comment (leaanthony) [inline cmd/linuxdb.yaml:321]: Just checking pkgconf-pkg-config is right

**Review by leaanthony — APPROVED** [review]

File: cmd/linuxdb.yaml (hunk 2 above)
Comment (leaanthony) [reply cmd/linuxdb.yaml:321]: Thanks for doing this! Will make sure `prt` gets added to v2!

Comment (alexmat) [reply cmd/linuxdb.yaml]: typo on my end, commited the fix, thanks for catching it

File: cmd/linuxdb.yaml (hunk 1 above)
Comment (alexmat) [reply cmd/linuxdb.yaml]: error on my end, fixed :)

**Comment by leaanthony** [comment]
Hi. Thanks for the updates. Can you please rebase this on the `develop` branch. Thanks!

**Comment by leaanthony** [comment]
@alexmat - Hi. Please rebase to the develop branch. Thanks!
